stetho/src/main/java/com/facebook/stetho/inspector/network/ResponseBodyFileManager.java

```diff
@@ -69,6 +79,13 @@ private static String readContentsAsUTF8(InputStream in) throws IOException {
     return out.toString("UTF-8");
   }
 
+  private String prettyPrintContent(InputStream in, AsyncPrettyPrinter asyncPrettyPrinter)
+      throws IOException {
+    ByteArrayOutputStream out = new ByteArrayOutputStream();
+    Util.copy(in, out, new byte[1024]);
```

Comment:
is this going to happen frequently? if so then we really need to pool these byte arrays to avoid GC churn. can defer on this for now, but if so then make a note to definitely tackle this later. and I think there's a few other places where we do this and we should be pooling there as well, so solving it here could provide a solution in those other places as well